pyaedt/edb_core/stackup.py

```diff
@@ -572,16 +576,31 @@ def import_stackup(self, fpath):
         return True
 
     @pyaedt_function_handler
-    def export_stackup(self, fpath, file_format="csv"):
-        """Export stackup definition to csv file.
+    def export_stackup(self, fpath, file_format="csv", include_material_with_layer=False):
+        """Export stackup definition to csv or json file.
```

Comment:
```suggestion
        """Export stackup definition to a CSV or JSON file.
```
When file extensions are used generically, they should be in all capital letters.